README.md

```diff
@@ -213,22 +215,53 @@ However in the following rare circumstances you might run into difficulties:
 
-The following are the notifications available as part of "deployments" feature:
-1. Deployment review pending notifications for your environments being deployed through GitHub Actions workflow.
-<p align="left"><img width="500" alt="Review Pending" src="docs/images/ReviewPending.png"></p>
+- You will get notified when a new workflow run is triggered. And you can track the live status of the jobs. 
+- You can track the approval notifications as a reply in the thread and you can approve the notifications directly from channel/personal app.
+- Once the workflow is completed, you will get a update as a reply in th thread so that you can complete context and history about the workflow run.
+- If something fails, you can choose to rerun the workflow in place and you can also enable debug logs if needed.
 
-2. Deployment review completed notifications for your environments being deployed through GitHub Actions workflow.
-<p align="left"><img width="500" alt="Approval completed" src="docs/images/ReviewCompleted.png"></p>
+#### Workflow notification filters
+Getting notified about each and every workflow run notification can be noisy. So, we are providing you capability to filter the notifications based on your requirement.
+You can filter your actions workflows notifications based on name, event, actor and/or branch. You can filter the notifications as below.
+
+`/github subscribe owner/repo workflows:{name:"your workflow name" event:"workflow event" branch:"branch name" actor:"actor name"}`
+
+- **name**: Name of your workflow
+- **event**: The event on which the workflow is triggered. You can find all the available events list [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#available-events).
+- **actor**: The person who triggered or responsible for running of the workflow
+- **branch**: The branch on which the workflow is running. Only incase where pull_request event is included, the branch will be the target branch the pull request is created for.
+
+You can pass multiple entries for each of the events in as a comma separate list as below example:
+`/github subscribe org/repo workflows:{event:"pull_request","push" branch:"main","dev" actor:"ashokirla"}`
+
+By default, when you configure workflow notifications without passing any filters, it is configured for workflows triggered via pull requests targeting your default branch.
+You can pass one or multiple entries. 
+
+You can unsubscribe for workflows notifications by just running the below command:
+`/github unsubscribe org/repo workflows`
+
+#### Updates to the app permissions
+To receive the above notifications you need to grant access to receive Actions events by GitHub app in Slack. You will be prompted to do so as you try to subscribe for `workflows` feature for your organization the first time.
+
+<p align="left"><img width="500" alt="Workflow notification" src="docs/images/ActionsPermissions.png"></p>
```

Comment:
Hi, I've just received the ActionsPermissions message.
I understand that I need read permissions, but I don't understand why I need write permissions. Could you please clarify this point?

Reply:
I'm guessing the action buttons require write access.

<img width="367" alt="Screenshot 2022-12-02 at 13 49 08" src="https://user-images.githubusercontent.com/7644519/205296619-dc813d2c-ea19-4347-b899-bd37e45c5f58.png">

Reply:
@ashokirla Is it right?

Reply:
@shogo82148 Yes, the app needs write permissions to do certain actions on your behalf. Like Re-run all jobs, Approve/Reject, Cancel workflow, Re-run failed jobs.

Reply:
I would prefer to be able to opt in to the write-dependent features